### PR TITLE
added global info and collapsed nav bar for smaller viewports

### PR DESCRIPTION
**Pull request by MartinezBaker**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -44,67 +68,189 @@ const lightTheme = {
...
+    };
+    window.addEventListener("resize", updateMedia);
+    return () => window.removeEventListener("resize", updateMedia);
+    //eslint-disable-next-line
+  }, []);
   useEffect(() => {
     props.getGlobalInfo()
     //eslint-disable-next-line
   }, [])
+  console.log(isScreen)
```
Comment (yoloOnTheBattlefield) [inline src/App.js]: no console

File: src/App.js [hunk 2]
```diff
@@ -44,67 +68,189 @@ const lightTheme = {
...
+                  <Link
+                    style={{
+                      textDecoration: "none",
+                      color: props.main.darkMode ? "white" : "black",
+                      fontSize: "14px"
+                    }}
+                    to="/coins"
+                  >
+                    Coins
+                  </Link>
```
Comment (yoloOnTheBattlefield) [inline src/App.js]: you can also style this using styled components

File: src/App.js [hunk 3]
```diff
@@ -44,67 +68,189 @@ const lightTheme = {
...
+            </GlobalInfoFour>
+            <StyledImgEth alt="Coin" src={props.coins.coins[1]?.image} />
+            <GlobalInfoFive>
+              {formatPercent(
+                props.main.globalInfo.data?.market_cap_percentage.eth.toFixed(0)
+              ).toString()}{" "}
+              <ProgressBar
+                progress={props.main.globalInfo.data?.market_cap_percentage.eth}
+                width={"50px"}
+                height={"13px"}
```
Comment (yoloOnTheBattlefield) [inline src/App.js:197]: do you use this progress bar with a different height? if no, consider removing this prop

File: src/pages/Coins/Coins.js [hunk 4]
```diff
@@ -32,13 +32,13 @@ const Coins = ({main, getChartData, getCoins, coins, getMoreCoins, sortItems, so
...
   useEffect(() => {
-    if(window.innerWidth > 1020){
+    if(window.innerWidth > 900){
       setScreen(true)
     } else {
       setScreen(false);
     }
     const updateMedia = () => {
-      if (window.innerWidth > 1020) {
+      if (window.innerWidth > 900) {
```
Comment (yoloOnTheBattlefield) [inline src/pages/Coins/Coins.js]: i see you're using this screen size functionality in multiple spots, consider making a reusable hook

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 3 above)
Comment (MartinezBaker) [reply src/App.js:197]: I switch the height in other areas but I also have a default height

**Review by yoloOnTheBattlefield — APPROVED** [review]

**Pull request merged**